ui/components/MarkdownBlock/index.js

```diff
@@ -7,3 +8,7 @@ export default function MarkdownBlock({ md }) {
     </div>
   );
 }
+
+export const MarkdownRender = ({ md }) => (
+  <div dangerouslySetInnerHTML={{ __html: parse(md) }} />
```

Comment:
Why would you use this and not `MarkdownBlock`?

Reply:
`react markdown` promises to render tables with the addition of plugins but seems to struggle with it in reality. I backed away from using it for anything other than simple inline markdown rendering. We're also using it for `Snippet` so I kept it around for now

Reply:
So the content from CKAN is mostly markdown, but with some inline html?

I'm just deeply, deeply wary of letting `dangerouslySetInnerHTML` into a codebase without some tight controls over the input (which seem absent here, due to the need to support arbitrary html in the markdown).

Reply:
I feel that's totally reasonable again! I can wrap the `dangerouslySetInnerHTML` in a `isomorphic-dompurify` for now until we come up with something better. Does that work?

Reply:
That is probably fine, the only change I might make would be to potentially move all the markdown parsing and sanitising to the server in the "get props" function, so the prop ends up being the sanitised html string, that way it can be done once at build time and avoids the need to send extra libraries to the client to do the work again there.

Reply:
Yeah I think that works, especially as we're rendering a static page. Will have a go now